test/llvm_test_code/TaintConfig/AttrConfig/basic_04.c

```diff
@@ -0,0 +1,6 @@
+int main() {
+  int I = 42;
+  __attribute__((annotate("psr.source"))) int J = 13;
```

Comment:
The `[[clang::annotate("...")]]` attribute syntax is C++-only. Since these test files contain plain C code, we need to use C-compliant attribute syntax